src/libraries/System.Security.Cryptography.Encoding/tests/Cbor/CborInitialByte.cs

```diff
@@ -6,6 +6,14 @@
 
 namespace System.Security.Cryptography.Encoding.Tests.Cbor
 {
+    internal enum CborConformanceLevel
+    {
+        NonStrict = 0,
```

Comment:
Accepting suggestions on a better name for `NonStrict`, which is just the current conformance behaviour of the implementation. The only difference from strict mode as defined in the spec is that it doesn't check for key uniqueness in maps.

Reply:
Casual? Permissive?

Reply:
Usually the converse of `strict` is `lax`.

Reply:
So `Lax`? Or `LaxConformance`?

Reply:
I'd say just `Lax` since `Conformance` is already in the enum name.

Reply:
> Usually the converse of strict is lax.

Depends on context I guess, "non-strict evaluation" is a commonly accepted term for example.